src/MusicalScore/Graphical/MusicSheetDrawer.ts

```diff
@@ -417,6 +419,28 @@ export abstract class MusicSheetDrawer {
                 this.drawLabel(label, <number>GraphicalLayers.Notes);
             }
         }
+        // Draw bounding boxes for debug purposes. This has to be at the end because only
+        // then all the calculations and recalculations are done
+        if (process.env.DRAW_BOUNDING_BOXES === "1") {
+            this.drawBoundingBoxes(page.PositionAndShape, 0, process.env.BOUNDING_BOX_TYPE);
+        }
+
+    }
+
+    private drawBoundingBoxes(startBox: BoundingBox, layer: number = 0, type: string = undefined): void {
```

Comment:
You might want to add a docstring explaining the parameters, as they are not self-explanatory. At least to me.